frontend/src/app/containers/HomePageNoAddon.js

```diff
@@ -30,7 +30,9 @@ export default class HomePageNoAddon extends React.Component {
   props: HomePageNoAddonProps
 
   render() {
-    const { experiments, isAfterCompletedDate, featuredExperiments } = this.props;
+    const { isAfterCompletedDate, featuredExperiments } = this.props;
+
+    const experiments = this.props.experiments.filter(x => !x.is_featured);
```

Comment:
This should be in a [selector](https://github.com/mozilla/testpilot/blob/master/frontend/src/app/selectors/experiment.js) (see: [`reselect`](https://github.com/reactjs/reselect)) to centralize this type of logic and prevent unnecessary recalculation and rendering of the DOM.